README.md

```diff
@@ -5,13 +5,22 @@
 [![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=Aalto-LeTech_intellij-plugin&metric=alert_status)](https://sonarcloud.io/dashboard?id=Aalto-LeTech_intellij-plugin)
 [![JetBrains IntelliJ plugins](https://img.shields.io/jetbrains/plugin/d/13634-a-plugin-for-intellij?label=plugin%20downloads)](https://plugins.jetbrains.com/plugin/13634-a-plugin-for-intellij)
 
-Intellij IDEA plugin for A+
+# ![Small_Logo](images/small_logo.png) Intellij IDEA plugin for A+ Courses
 
-## Styling
+## Maintenance
 
-This project uses slightly modified google checkstyle rules from [Checkstyle GitHub](https://github.com/checkstyle/checkstyle/blob/checkstyle-8.12/src/main/resources/google_checks.xml) for code formatting. The particular version applied to this project is stored at: `checkstyle/`. Please note, that the checkstyle file itself is licensed under **GNU LGPL** (stored at that same folder). Scala code is checked using the [default rules from the scalastyle repository](https://github.com/scalastyle/scalastyle/blob/master/src/main/resources/default_config.xml). The configuration file is located in `scalastyle` and it is licensed under the **Apache-2.0** license.  
+If you are a Student discovered an issue you'd like to report, please turn to [Teaching Assistants (TA)](https://plus.cs.aalto.fi/o1/2020/w01/ch01/#course-staff), so they would handle. For (head) TAs, please create issues directly here: [Aalto-LeTech/intellij-plugin/issues](https://github.com/Aalto-LeTech/intellij-plugin/issues).
+
+Once the issue (bug report) is made, the development team (**[@OlliKiljunen](https://github.com/OlliKiljunen)**, **[@nikke234](https://github.com/nikke234)**, **[@superseacat](https://github.com/superseacat)**) will handle it on a **best-effort basis:**
```

Comment:
We could refer to the team https://github.com/orgs/Aalto-LeTech/teams/idea-plugin-dev instead of listing developers here.

Reply:
users outside Aalto-LeTech and not-logged-ins can't see that page :disappointed:

Reply:
I wonder if listing the team members here is necessary at all? I don't have a strong opinion on this though